source/backends/debian/debpkgindex.d

```diff
@@ -112,7 +114,9 @@ public:
                                             "Translation-%s.%s".format(lang, "%s"));
 
             try {
-                fname = downloadIfNecessary (rootDir, tmpDir, fullPath);
+                synchronized (this) {
+                    fname = downloadIfNecessary (rootDir, tmpDir, fullPath);
```

Comment:
Can we make this to only synchronize if we actually download anything? Because otherwise this will slow down things even if the file is already downloaded and present on disk.